docs/install_guides/installing-model-dev-tools.md

```diff
@@ -8,7 +8,89 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
 * Post-Training Optimization Tool
 * Model Downloader and other Open Model Zoo tools
 
-From the 2022.1 release, OpenVINO Development Tools can only be installed via PyPI.
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. 
+
+## For Python Developers
+
+If you are a Python developer, you can find the main steps below to install OpenVINO Development Tools. For more details, see <https://pypi.org/project/openvino-dev>.
+
+While installing OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately.
+
+### Step 1. Set Up Python Virtual Environment
+
+To avoid dependency conflicts, use a virtual environment. Skip this step only if you do want to install all dependencies globally.
+
+Use the following command to create a virtual environment:
+
+@sphinxdirective
+
+.. tab:: On Linux and macOS
+
+   .. code-block:: sh
+   
+      # Depending on your operating system, this step may require installing python3-venv
+      python3 -m venv openvino_env
```

Comment:
#comments should not be a part of code block, since it is included in a buffer by clicking at copy button.
```suggestion
      python3 -m venv openvino_env
```